packages/core/test/request-builder/odata-request.spec.ts

```diff
@@ -127,6 +127,24 @@ describe('OData Request', () => {
       requestSpy.mockRestore();
     });
 
+    it('request config contains headers without etag value when there is no etag config', async () => {
+      const expectedConfigEntry = {
+        headers: {
+          accept: 'application/json',
+          'content-type': 'application/json'
+        }
+      };
+      const destination: Destination = {
+        url: 'http://example.com'
+      };
+
+      await createRequest(ODataGetAllRequestConfig, destination).execute();
+
+      expect(axios.request).toHaveBeenCalledWith(
+        expect.objectContaining(expectedConfigEntry)
+      );
+    });
```

Comment:
This does not test that there is no eTag value in the header, if I am not mistaken. You could make this more explicit by checking that `axios.request.headers` does not contain the `if-match` key.